### PR TITLE
Test1

### PR DESCRIPTION
**Pull request by hruy28**
We should also change the other images.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -29,10 +29,10 @@ And we converted this pdf data to json file and so we have these words in json f
 ### Wireframe
 
 **Home Page**
-![Home Page](/doc/UserInteraction_00.jpg)
+![Home Page](public/images/frontEnd.jpg)
 
 **User Flow**
-![User Flow](/doc/UserInteraction_01.jpg)
```
Comment (hruy28) [inline Readme.md:35]: You should also delete these one.

File: Readme.md [hunk 2]
```diff
@@ -29,10 +29,10 @@ And we converted this pdf data to json file and so we have these words in json f
...
+![User Flow](/images/frontEnd.jpg)
```
Comment (gidesegid) [inline Readme.md:35]: do i need to delete this as well

**Review by hruy28 — COMMENTED** [review]
I have commented on line 32. Just test.

File: Readme.md (hunk 2 above)
Comment (hruy28) [reply Readme.md:35]: Yes.

**Pull request merged**